docs-gen/hugo.toml

```diff
@@ -39,5 +34,13 @@ canonifyURLs = true
   github_branch = "master"
   github_project_repo = "https://github.com/sky-uk/kfp-operator"
 
+[params.ui]
+navbar_logo = false
+showLightDarkModeMenu = true
+
 [security.exec]
   osEnv = ['(?i)^((HTTPS?|NO)_PROXY|PATH(EXT)?|APPDATA|TE?MP|TERM|HOME|ASDF_DIR)$']
+
+[markup.goldmark.renderer]
+unsafe = true
```

Comment:
For anyone wondering:

This doesn't inherently make the website unsafe. It just allows for the rendering of <html> elements from within markdown files.

Reply:
yup, it's required now if you want to mix HTML and Markdown in the same file. It's described as 'unsafe' as it could allow others to execute JS from within the HTML if we were passing in user generated content, but as we control all of the content here, it's fine imo

Reply:
See https://gohugo.io/getting-started/configuration-markup/#rendererunsafe